src/style.css

```diff
@@ -0,0 +1,125 @@
+/* standart */
+
+@font-face {
+  font-weight: 300;
+  font-family: Avenir;
+  src: url(./fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-weight: 400;
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-weight: 900;
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.form__our {
+  position: relative;
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  margin-top: 20px;
+  border-radius: 4px;
+  background-color: #fff;
+  box-shadow: 0 1px 8px rgb(61 78 97 / 10%);
+}
+
+.form__our:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+/* big */
+
+.label__big {
+  height: 68px;
+  display: block;
+  width: 100%;
+}
+
+.input__big {
+  padding-left: 62px;
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  width: 100%;
+  height: 100%;
+  border: none;
+  outline: none;
+  font-weight: 900;
+}
+
+.input__big::placeholder {
+  font-family: Avenir, sans-serif;
+  font-weight: 400;
+  font-size: 16px;
+}
+
+.input__big:focus {
+  border-radius: 3px;
+  color: #3d4e61;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+.button__big {
+  width: 62px;
+  height: 100%;
+  position: absolute;
+  left: 0;
+  border: none;
+  outline: none;
+  background: url(./images/Search.svg) no-repeat 50%;
+  background-color: transparent;
+  cursor: pointer;
```

Comment:
you repeat these styles in button_small, move them to one class that you use for both elements